dcpy/lifecycle/package/validate.py

```diff
@@ -8,13 +9,20 @@
 from dcpy.utils.logging import logger
 
 
-class Errors:
+class ErrorType:
     INVALID_DATA = "INVALID_DATA"
     NULLS_FOUND = "NULLS_FOUND"
     COLUMM_MISMATCH = "COLUMN_MISMATCH"
     INVALID_METADATA = "INVALID_METADATA"
 
 
+@dataclass
+class ValidationError:
+    error_type: ErrorType
```

Comment:
Blanking on how simple classes like this work in python - should `ErrorType` be an enum? Or will this work as written

Reply:
Is logic for `Enum` classes more around conversion to their types? Like having a built-in `.value` function

Reply:
Apologies, this was sloppy commiting. ErrorType is enum'd in a later commit